apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -347,6 +349,20 @@ export default function manageStudents(state=initialState, action) {
       }
     };
   }
+  if (action.type === EDIT_ALL) {
+    let newState = {
+      ...state
+    };
+    for (const studentKey in state.studentData) {
+      const student = state.studentData[studentKey];
+      newState.studentData[student.id].isEditing = true;
+      newState.editingData[student.id] = {
+        ...newState.studentData[student.id],
+        ...state.editingData[student.id],
```

Comment:
It would be helpful to have a brief comment on initialState explaining the relationship between isEditing, studentData, and editingData. If my understanding is correct, then it could just say that studentData represents the data about the student that's already been saved to the server, and if isEditing is true then editingData represents the data in the edit fields on the client which has not yet been persisted to the server.

Reply:
You're correct. I'll add a comment in a follow up PR.